substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/Range.java

```diff
@@ -133,6 +133,10 @@ public String getFullMethodName() {
         return fullMethodName;
     }
 
+    public boolean isDeoptTarget() {
+        return methodName.endsWith("**");
```

Comment:
Might be a good idea to define a public static field `METHOD_NAME_DEOPT_SUFFIX` for `"**"` in `src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/meta/HostedMethod.java:247` and use that field also here.

Reply:
Yes, will do.